### ISSUE TITLE
No bars in terminal emulator

### ISSUE THREAD

**Issue by LuxLucian** [karlstav/cava#399]
Running cava in a terminal (alacritty, xterm, urxvt) shows no bars. The background can be changed, and running it in a TTY shows the bars. I did try the noncurses and raw modes, and I am on Arch.

**Comment by LuxLucian** [comment]
I forgot to mention I am using the pulseaudio monitor.

**Comment by karlstav** [comment]
can this be the same as #109 and #121?

I thought I fixed that...

try setting your locale as described in those issues.

**Comment by LuxLucian** [comment]
I run the command with no errors prior, but I will try it again. Do you
think running cava on Wayland might have any effect?

**Comment by LuxLucian** [comment]
LC_ALL=en_US.UTF-8 cava
Solved the issue!

**Comment by karlstav** [comment]
that's strange, that bug should have been fixed in d4848c3ebd57fc14d4510a97285b39df10aa2570

@LuxLucian can I ask you what version are you on? `cava -v`

**Comment by nekoize** [comment]
> LC_ALL=en_US.UTF-8 cava
> Solved the issue!

i have the same issue, but this didn't do anything. 
cava -v output is 0.7.3 if so

**Comment by nekoize** [comment]
ah nevermind,  my locale.conf is configured wrong
make sure you adjust your locale.conf to your liking
```
LC_MESSAGES=en_US.UTF-8
LANGUAGE=en_US
LANG=en_US.UTF-8
```

**Comment by ThreshMain** [comment]
Same issue.
LC_ALL=en_US.UTF-8 cava
Solved the issue!
cava -v output is 0.7.3 if so

**Comment by karlstav** [comment]
hi @ThreshMain,

out of curiosity, what is the output of the command `locale` in your terminal?

I am just trying to recreate this issue.

**Comment by notkirito** [comment]
I also have this issue, and `LC_ALL=en_US.UTF-8 cava` fixed it. Maybe I'm suffering from the same problem? Here's the output of `locale`:
```
LANG=C
LC_CTYPE=
LC_NUMERIC="C"
LC_TIME="C"
LC_COLLATE="C"
LC_MONETARY="C"
LC_MESSAGES="C"
LC_PAPER="C"
LC_NAME="C"
LC_ADDRESS="C"
LC_TELEPHONE="C"
LC_MEASUREMENT="C"
LC_IDENTIFICATION="C"
LC_ALL=
```

**Comment by notkirito** [comment]
Yep, it was my /etc/locale.conf that didn't exist. I can confirm that it's a locale problem.

**Comment by karlstav** [comment]
I'll just leave this open until I have a fix or proper description in readme

**Comment by cookstown** [comment]
> LC_ALL=en_US.UTF-8 cava Solved the issue!

just had the same issue, fixed it, ty

**Comment by SiwyKot41** [comment]
> LC_ALL=en_US.UTF-8 cava Solved the issue!
@LuxLucian so much thank you, you are solved my problem :D

**Comment by docentYT** [comment]
`LC_ALL=en_US.UTF-8 cava` did not help me. The `/etc/locale.conf` file I had. I had to regenerate the locale and set `/etc/environment` to
```
LANG=en_US.utf-8
LC_ALL=en_US.utf-8
```

**Comment by n6v26r** [comment]
I also have this issue after the latest update.
